app/models/conversion_host.rb

```diff
@@ -274,16 +283,46 @@ def miq_ssh_util_args_manageiq_providers_openstack_cloudmanager_vm
   # +extra_vars+ option should be a hash of key/value pairs which, if present,
   # will be passed to the '-e' flag.
   #
-  def ansible_playbook(playbook, extra_vars = {})
-    command = "ansible-playbook #{playbook} -i #{ipaddress}"
+  def ansible_playbook(playbook, extra_vars = {}, auth_type = nil)
+    task = MiqTask.all.select { |t| t.context_data.present? && t.context_data[:conversion_host_id] == id }.sort_by(&:created_on).last
 
-    extra_vars[:v2v_host_type] = resource.ext_management_system.emstype
-    extra_vars.each { |k, v| command += " -e '#{k}=#{v}'" }
+    host = hostname || ipaddress
 
-    connect_ssh { |ssu| ssu.shell_exec(command) }
+    command = "ansible-playbook #{playbook} --inventory #{host}, --become --extra-vars=\"ansible_ssh_common_args='-o StrictHostKeyChecking=no'\""
```

Comment:
I don't have an issue with this, but AwesomeSpawn does a really nice job of building this type of command line, check out https://github.com/ManageIQ/awesome_spawn/blob/master/lib/awesome_spawn/command_line_builder.rb#L8-L43 if you want to clean this up in a follow-up